prody/dynamics/analysis.py

```diff
@@ -646,6 +646,143 @@ def identify(v):
         return sorted(set(hinge_list))
     return hinges
 
+
+
+def hingeFinder(gnm, n_modes=None, threshold=15, space=None, trim=False):
+    """
+    [HZ384] H Zhang, M Gur, I Bahar (2024) Global hinge sites of proteins as target sites for drug binding Proc Natl Acad Sci USA 121 (49), e2414333121 
+    Updated hinge identification based on [HZ384]. This will:
+        1) If GNM model is provided without specification of n_modes, number of modes will be determined to achieve 33% cumulative variance.
```

Comment:
I guess a different percentage could also be used as a decimal input to the number of modes